test_hyperparameter_clustering.py

```
@@ -0,0 +1,341 @@
+from app.services.review_analyzer import _embed_sentences, analyze_reviews
```

Ruff (0.12.2)

`app.services.review_analyzer._embed_sentences` imported but unused

Remove unused import

(F401)